readme.md

```diff
@@ -2517,6 +2517,18 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 |-------: | -------: | ------:|
 |   2 - 5 |       13 |  90m   |
 
+### [The Witcher: Path Of Destiny](https://boardgamegeek.com/boardgame/401325/witcher-path-destiny)
+
+>Take on the role of Geralt, Yennefer, Ciri, Vesemir, or Dandelion and retell the iconic stories from the saga.
+Use your unique abilities and follow the path of destiny or completely change its course. The actions taken by the group will determine how the story ends - but only one of you will be remembered as a hero....
```

Comment:
The failing linter message cites what’s wrong, the quote block needs to be updated.

```suggestion
> Use your unique abilities and follow the path of destiny or completely change its course. The actions taken by the group will determine how the story ends - but only one of you will be remembered as a hero....
```